Documentation/cmdref/cilium-agent.md

```diff
@@ -81,7 +82,7 @@ cilium-agent [flags]
       --container-ip-local-reserved-ports string                  Instructs the Cilium CNI plugin to reserve the provided comma-separated list of ports in the container network namespace. Prevents the container from using these ports as ephemeral source ports (see Linux ip_local_reserved_ports). Use this flag if you observe port conflicts between transparent DNS proxy requests and host network namespace services. Value "auto" reserves the WireGuard and VXLAN ports used by Cilium (default "auto")
       --controller-group-metrics strings                          List of controller group names for which to to enable metrics. Accepts 'all' and 'none'. The set of controller group names available is not guaranteed to be stable between Cilium versions.
       --crd-wait-timeout duration                                 Cilium will exit if CRDs are not available within this duration upon startup (default 5m0s)
-      --datapath-mode string                                      Datapath mode name (veth, netkit, netkit-l2, lb-only) (default "veth")
+      --datapath-mode string                                      Datapath mode name (veth, netkit, netkit-l2) (default "veth")
```

Comment:
This probably warrants an upgrade note for whoever was using `--datapath-mode` to setup `lb-only` mode?

Reply:
ack will add

Reply:
done